_data/engine-cli-edge/docker_stack.yaml

```diff
@@ -16,7 +16,28 @@ clink:
 - docker_stack_ps.yaml
 - docker_stack_rm.yaml
 - docker_stack_services.yaml
+options:
+- option: kubeconfig
+  value_type: string
+  description: Kubernetes config file
+  deprecated: false
+  experimental: false
+  experimentalcli: true
+  kubernetes: true
+  swarm: false
+- option: namespace
+  value_type: string
+  default_value: default
+  description: Kubernetes namespace to use
+  deprecated: false
+  experimental: false
+  experimentalcli: true
+  kubernetes: true
+  swarm: false
 deprecated: false
 min_api_version: "1.25"
 experimental: false
+experimentalcli: false
+kubernetes: false
+swarm: false
```

Comment:
I notice the `stack` commands themselves are not annotated with "swarm" or "kubernetes"; opening a pull request for that for a follow up.

Reply:
Opened https://github.com/docker/cli/pull/804 for this